src/components/popular-categories/PopularCategories.styles.ts

```diff
@@ -1,16 +1,20 @@
 export const styles = {
   wrapper: {
-    margin: { xs: '40px 0', sm: '75px 0', md: '100px 0' },
-    minHeight: '500px'
+    textAlign: 'center',
```

Comment:
the same as above, check which styles can be left as default to not ruin another components based on this one, and other style that you need you should pass as prop